templates/patient_file/transactions/list.html.twig

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>{{ 'Patient Transactions'|translate }}</title>
+    {{ header|raw }}
```

Comment:
see my comment above regarding a function to call the class (that doesn't exist yet) here.